course-definition.yml

```
@@ -676,4 +687,96 @@ stages:
     primary_extension_slug: "pub-sub"
     name: "Unsubscribe"
     difficulty: medium
-    marketing_md: In this stage, you'll add support for the `UNSUBSCRIBE command`, which is used to unsubscribe from a channel.
\ No newline at end of file
+    marketing_md: In this stage, you'll add support for the `UNSUBSCRIBE command`, which is used to unsubscribe from a channel.
+
+  # Sorted Sets
+  - slug: "ct1"
+    primary_extension_slug: "zsets"
+    name: "Create sorted set"
+    difficulty: easy
+    marketing_md: |
+      In this stage, you'll add support for creating a new sorted set using the `ZADD` command.
+
+  - slug: "hl2"
+    primary_extension_slug: "zsets"
+    name: "Validate score values"
+    difficulty: easy
+    marketing_md: |
+      In this stage, you'll add support for verifying the value of the score argument for `ZADD` command.
+
+  - slug: "hf1"
+    primary_extension_slug: "zsets"
+    name: "Add to existing zset"
```

```suggestion
    name: "Add members"
```

(The zset context is already present, doesn't need to be repeated)